### PR TITLE
[qosorch]: Remove Init Color ACLs

### PR DESCRIPTION
**Pull request by zhenggen-xu**
Remove Init Color ACLs

InitSystemAcl table is removed.
ECN ACL rules: DSCP 0/8 mapping to yellow are removed. 

Signed-off-by: Zhenggen Xu <zxu@linkedin.com>

**What I did**
InitSystemAcl table is removed.
ECN ACL rules: DSCP 0/8 mapping to yellow are removed. 

**Why I did it**
The default DSCP mapping maps DSCP 0/8 to Yellow. This is not desired as some platforms would treat those DSCP differently in terms of drop precedence of TX scheduling and buffer thresholds etc.

The DSCP mapping should be done explicitly by configurations like buffer profiles etc.

**How I verified it**
After I remove the mapping, the ACL rules were gone. 
The yellow drop counters on my DUT were gone, and was changed to regular drop counter.

**Details if related**

### CODE REVIEW

File: orchagent/qosorch.cpp [hunk 1]
```diff
@@ -760,10 +760,6 @@ void QosOrch::initColorAcl()
     // init ACL system table
     acl_table_id = initSystemAclTable();
 
-    // Add entry to match packets with dscp=8, ecn=0 and set yellow color to them
-    initAclEntryForEcn(acl_table_id, 1000, 0x00, 0x08, SAI_PACKET_COLOR_YELLOW);
-    // Add entry to match packets with dscp=0, ecn=0 and set yellow color to them
-    initAclEntryForEcn(acl_table_id,  999, 0x00, 0x00, SAI_PACKET_COLOR_YELLOW);
```
Comment (lguohan) [inline orchagent/qosorch.cpp]: suggest to remvoe line 743

Comment (zhenggen-xu) [reply orchagent/qosorch.cpp]: done

**Comment by lguohan** [comment]
retest this please

**Comment by lguohan** [comment]
retest this please

**Comment by zhenggen-xu** [comment]
retest this please

**Comment by zhenggen-xu** [comment]
retest this please

**Comment by lguohan** [comment]
retest this please

**Comment by lguohan** [comment]
retest this please

**Comment by lguohan** [comment]
retest this please

**Comment by zhenggen-xu** [comment]
retest this please

**Comment by lguohan** [comment]
retest this please

**Pull request merged**